lib/core/presentation/pages/quest/quest_listing_page/widgets/quest_item_widget.dart

```diff
@@ -0,0 +1,139 @@
+        child: Container(
+          decoration: BoxDecoration(
+            color: LemonColor.atomicBlack,
+            borderRadius: BorderRadius.circular(LemonRadius.medium),
+          ),
+          clipBehavior: Clip.hardEdge,
+          child: Column(
+            children: [
+              Padding(
+                padding: EdgeInsets.all(Spacing.small),
+                child: Row(
+                  children: [
+                    Expanded(
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            title,
+                            style: Typo.small.copyWith(
+                              color: colorScheme.onSecondary,
+                              fontWeight: FontWeight.w400,
+                            ),
+                          ),
+                          SizedBox(
+                            height: 3.h,
+                          ),
+                          subTitle == ''
+                              ? const SizedBox.shrink()
+                              : Text(
+                                  subTitle,
+                                  style: Typo.medium.copyWith(
+                                    color: colorScheme.onPrimary,
+                                    fontWeight: FontWeight.w500,
+                                  ),
+                                  maxLines: 2,
+                                ),
+                        ],
+                      ),
+                    ),
+                    SizedBox(width: Spacing.small),
+                    Assets.icons.icArrowBack.svg(
+                      width: 18.w,
+                      height: 18.w,
+                    ),
+                  ],
+                ),
+              ),
+              repeatable == true
+                  ? Container(
+                      color: LemonColor.white03,
+                      padding: EdgeInsets.all(Spacing.small),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          Row(
+                            children: [
+                              Assets.icons.icRepeatRounded.svg(
+                                width: 18.w,
+                                height: 18.w,
+                              ),
+                              SizedBox(width: Spacing.superExtraSmall),
+                              Text(
+                                t.quest.repeatable,
+                                style: Typo.small.copyWith(
+                                  color: colorScheme.onSecondary,
+                                  fontWeight: FontWeight.w400,
+                                ),
+                              ),
+                            ],
+                          ),
+                          (trackingCount ?? 0) > 0
```

Comment:
```suggestion
if((trackingCount ?? 0) > 0) Widget()
```
No need to use ternary operator to render redundant widget